### ISSUE TITLE
[content idea] a11y screen reader and voice control tips for specific HTML features

### ISSUE THREAD

**Issue by fstanis** [ampproject/amp.dev#3218]
While testing AMP on iOS, we came across some browser-specific a11y issues, for example with `<input type="month">`, it can't be focused when using VoiceOver.

While this is not an issue with AMP per se, I think it would be valuable to mention somewhere (either in `<amp-form>` docs or a separate article) that input types like this should be avoided to prevent a11y issues.

I found a great website that tests and lists all of these issues in different browsers - might be worth investigating what other things have poor cross-browser support currently: https://a11ysupport.io/tests/

**Comment by sebastianbenz** [comment]
Great idea! One question would be whether we should add an accessibility section to relevant component doc pages or whether we should add a dedicated guide?

This should also fit into @caroqliu's work.

//cc @CrystalOnScript

**Comment by caroqliu** [comment]
Thanks for suggesting this, Filip! I believe both a dedicated guide and component-specific documentation would be useful. Something like what Filip is proposing would probably be a two-step change: (1) we could add an "Accessibility Considerations" or similar under Documentation > Guides and Tutorials > Develop that provides publishers with resources for a11y testing their websites with assistive technologies and make sure focus isn't being trapped, and (2) modify `amp-form` documentation to reference the central resource from (1) while also noting this specific gotcha.
